.github/workflows/coverage-report.yml

```diff
@@ -7,6 +7,9 @@ on:
     - "test/**"
     - "!examples/landmarks/**"
 
+permissions:
+  issues: write
```

Comment:
The docs are a little light, so I'm not sure if it also needs `content: read` for the clone operation